website/app/styles/app.scss

```diff
@@ -27,11 +27,13 @@
 
 html {
   box-sizing: border-box;
```

Comment:
Just curious, why add this style on just the html element vs. using a universal selector?

Reply:
Yes, you're right (I think I copied the code from somewhere, one of our repos, but now that I look at it doesn't make sense).